projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -366,6 +374,10 @@ function get_element_styles_from_attributes( $attributes ) {
  * @return string
  */
 function render_block( $attributes ) {
+	// If the Subscriptions module is not active, don't render the block.
+	if ( ! Jetpack::is_module_active( 'subscriptions' ) ) {
```

Comment:
Is it easily possible to add a test for this? If it is, it would be nice to add one.